app/services/platform-apps/index.ts

```diff
@@ -178,11 +178,6 @@ export class PlatformAppsService extends StatefulService<IPlatformAppServiceStat
 
     if (!this.userService.isLoggedIn()) return;
 
-    this.SET_DEV_MODE(await this.getIsDevMode());
-
-    this.loadProductionApps();
-    this.SET_APP_STORE_VISIBILITY(await this.fetchAppStoreVisibility());
-
```

Comment:
not really sure why this got removed?

Reply:
Having this code here gives a chance to run `loadProductionApps` twice. First time when the service has been itinitalized and the second time when we get the `loggedIn` event. BTW just reviewed and realized that I need to add more changes here

Reply:
fixed